quarkus/config-api/src/main/java/org/keycloak/config/StorageOptions.java

```diff
@@ -333,6 +334,14 @@ public String getProvider() {
             .description("Root directory for file map store.")
             .build();
 
+    public static final Option<String> STORAGE_JPA_DB = new OptionBuilder<>("storage-jpa-db", String.class)
+            .category(OptionCategory.STORAGE)
+            .defaultValue(Database.Vendor.POSTGRES.name().toLowerCase())
+            .expectedValues(Database::getAvailableMapStoreAliases)
+            .description("The database vendor for jpa map storage.")
+            .buildTime(true)
+            .build();
```

Comment:
I have a bit silly question. Why do we even need this new option? :) Can't we simply piggyback on `--db`?

I understand that this is a new Store option. But what will happen with the `--db` after the new Store replaces the legacy Store? Would we keep using `--storage-jpa-db`? Isn't it a bit long? :)

Reply:
At this moment there is a difference in "expected values' as well as in "default value" between `--db` and `--storage-jpa-db`. 

That's the reason it was introduced in the first place. 

I agree that after the new Store replaces the legacy Store it might be a bit long :) 

note: https://github.com/keycloak/keycloak/issues/13562

Reply:
The two options have a different set of possible values. All storage options are considered preview anyway, so I'm ok with having an extra option for now. Once we have per-realm-configuration of storages, this will soon look very different anyway.

Reply:
Makes sense.